### ISSUE TITLE
Fix uncontained <li> in Footer for a11y 

### ISSUE THREAD

**Issue by racedale** [Rebuild-Black-Business/RBB-Website#230]
Resolve issue from Lighthouse scan:
List items (<li>) are not contained within <ul> or <ol> parent elements.
